t/venom/pfservers/configurator/step1.yml

```diff
@@ -0,0 +1,112 @@
+name: step1 interfaces 
+version: 2
+testcases:
+- name: step1_dhcp_listener
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/interface/{{.configurator.dhcp_listener_interface}}'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "high_availability": 0,
+        "hwaddr": "{{.configurator.dhcp_listener_mac}}",
+        "id": "{{.configurator.dhcp_listener_interface}}",
+        "ifindex": "2",
+        "ipaddress": "{{.configurator.dchp_listener_ip}}",
+        "is_running": true,
+        "name": "{{.configurator.dhcp_listener_interface}}",
+        "netmask": "{{.configurator.dhcp_listener_mask}}",
+        "network_iseditable": false,
+        "type": "dhcp-listener"
+      }
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: step1_management_configuration
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/interface/{{.configurator_management_interface}}'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "additional_listening_daemons": ["portal"],
+        "0": "portal",
+        "address": "{{.configurator_management_interface_ip_and_mask}}",
+        "high_availability": 0,
+        "hwaddr": "{{.configurator_management_mac}}",
+        "id": "{{.configurator_management_interface}}",
+        "ifindex": "2",
+        "ipaddress": "{{.configurator_management_ip}}",
+        "is_running": true,
+        "name": "{{.configurator_management_interface}}",
+        "netmask": "{{.configurator_management_mask}}",
+        "network": "{{.configurator_management_network_ip}}",
+        "network_iseditable": false,
+        "type": "management"
+      }
```

Comment:
Same comment as previously, removed all items not necessary like:
- `is_running`
- `network_iseditable`
- `high_availability`
- `hwaddr`